data/json/overmap/overmap_special/specials.json

```
@@ -314,6 +314,19 @@
     "occurrences": [ 0, 2 ],
     "flags": [ "CLASSIC" ]
   },
+  {
+    "type": "overmap_special",
+    "id": "Cabin_Lapin",
+    "overmaps": [
+      { "point": [ 0, 0, 0 ], "overmap": "cabin_lapin_north" },
+      { "point": [ 0, 0, 1 ], "overmap": "cabin_roof_lapin_north" }
+    ],
+    "locations": [ "wilderness" ],
+    "city_distance": [ 20, -1 ],
+    "city_sizes": [ 0, 20 ],
+    "occurrences": [ 0, 2 ],
```

```suggestion
    "occurrences": [ 0, 1 ],
```
There's no need for two of the same guy showing up per overmap.